examples/knative/powershell/kn-ps-webhook/README.md

```diff
@@ -0,0 +1,175 @@
+
+* WEBHOOK_USERNAME - Username to login to webhook function endpoint
+* WEBHOOK_PASSWORD - Password to login to webhook function endpoint
+
+
+Start the container image by running the following command:
+
+```console
+docker run -e FUNCTION_DEBUG=true -e PORT=8080 --env-file docker-test-env-variable -it --rm -p 8080:8080 <docker-username>/kn-ps-webhook:${TAG}
+```
+
+In a separate terminal, run either `send-cloudevent-test.ps1` (PowerShell Script) or `send-cloudevent-test.sh` (Bash Script) to simulate a CloudEvent payload being sent to the local container image
+
+```console
+Testing Function ...
+See docker container console for output
+
+# Output from docker container console
+Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
+--     ----            -------------   -----         -----------     --------             -------
+2      Job2            ThreadJob       NotStarted    False           PowerShell           …
+09/19/2021 13:34:04 - PowerShell HTTP server start listening on 'http://*:8080/'
+09/19/2021 13:34:04 - Processing Init
+
+09/19/2021 13:34:04 - Init Processing Completed
+
+09/19/2021 13:34:09 - Processing Context ...
+
+09/19/2021 13:34:09 - [Function Secrets]:
+{"WEBHOOK_USERNAME": "XXX","WEBHOOK_PASSWORD": "XXX"}
+
+09/19/2021 13:34:09 - Checking for authorization ...
+
+09/19/2021 13:34:09 - [Expected Authorization Header]:
+Basic XXX
+
+09/19/2021 13:34:09 - [Headers]:
+
+Name                           Value
+----                           -----
+Content-Length                 {57}
+User-Agent                     {curl/7.64.1}
+Accept                         {*/*}
+SendToBroker                   {False}
+Host                           {localhost:8080}
+Authorization                  {Basic XXX}
+Content-Type                   {application/json}
+
+09/19/2021 13:34:09 - [Body]:
+@{VmName=Production-01; UserName=Michael Gasch}
+
+09/19/2021 13:34:09 - Creating custom CloudEvent ...
+
+09/19/2021 13:34:09 - Local developmen detected, will not forward custom CloudEvent to VMware Event Broker
+```
+
+# Step 3 - Deploy
+
+> **Note:** The following steps assume a working Knative environment using the
+`default` Rabbit `broker`. The Knative `service` and `trigger` will be installed in the
+`vmware-functions` Kubernetes namespace, assuming that the `broker` is also available there.
+
+Push your container image to an accessible registry such as Docker once you're done developing and testing your function logic.
+
+```console
+docker push <docker-username>/kn-ps-webhook:${TAG}
+```
+
+Update the `webhook_secret.json` file with your vCenter Server credentials and configurations and then create the kubernetes secret which can then be accessed from within the function by using the environment variable named called `WEBHOOK_SECRET`.
```

Comment:
The description feels like it's copy&paste and needs some adjustments for this example.